src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,91 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+        Driver driver = new Driver();
+        driver.setName("Bob");
+        driver.setLicenseNumber("12345");
+        driverService.create(driver);
+        System.out.println(driverService.get(23L));
```

Comment:
Don't hardcode IDs. For example, if I run your code on my PC, I'll have an exception (or Optional.empty()), because I'll have only the first driver in my DB. After CRUD operations use getAll methods. If you want to get/delete/update by ID use the list from getAll method. And log your operations with sout